tools/fds/server.h

```diff
@@ -89,7 +90,12 @@ const option::Descriptor usage[] = {
       "\t   listening on 172.30.144.1 with UDP port 12345 and provided with a\n"
       "\t   backup file. If the server crashes it will automatically restore its\n"
       "\t   previous state when reenacted.\n\n"
-      "\t$ " FAST_SERVER_BINARY " -i 1 -l 172.30.144.1 -p 12345 -b" },
+      "\t$ " FAST_SERVER_BINARY " -i 1 -l 172.30.144.1 -p 12345 -b\n\n"
```

Comment:
```suggestion
      "\t$ " FAST_SERVER_BINARY " -i 4 -l 172.30.144.1 -p 12345 -b\n\n"
```

Also, the previous example has the same errors as in the `README`:

```
      "\t$ " FAST_SERVER_BINARY " -i 3 -l 192.168.36.34 -p 8783 -l 172.20.96.1 -p 51083\n\n"
```